src/main/java/propertyFiles/Bundle.properties

```diff
@@ -482,6 +482,7 @@ harvestclients.newClientDialog.oaiSets.tip=Harvesting sets offered by this OAI s
 harvestclients.newClientDialog.oaiSets.noset=None
 harvestclients.newClientDialog.oaiSets.helptext=Selecting "none" will harvest the default set, as defined by the server. Often this will be the entire body of content across all sub-sets.
 harvestclients.newClientDialog.oaiSets.helptext.noset=This OAI server does not support named sets. The entire body of content offered by the server will be harvested.
+harvestclients.newClientDialog.oaiSets.listTruncated=Please note that the remote server was taking too long to return the full list of available OAI sets, so the list was truncated! Please select a set from the current list (or select the "no set" option), and try again later if you need to change it. 
```

Comment:
My only thought here is that we could replace the exclamation mark with a period. This is sort of a "voice" thing. I'm not sure what the voice for the Dataverse software is or should be.

Reply:
PLEASE NOTE THAT THE REMOTE SERVER WAS TAKING TOO LONG!!!!


![caps_on](https://user-images.githubusercontent.com/737558/75468329-0cced900-595b-11ea-9063-968265d744a4.jpg)

Reply:
ok, done.